.travis.yml

```diff
@@ -5,7 +5,7 @@ cache:
   directories:
     - ".meteor"
     - ".babel-cache"
-script: TEST_PACKAGES_EXCLUDE="less" ./packages/test-in-console/run.sh
+script: TEST_PACKAGES_EXCLUDE="less,blaze" ./packages/test-in-console/run.sh
```

Comment:
You're right, it's better to run these tests than to skip them. Merged your PR instead of this one. Thanks for getting to it before me!